.github/workflows/release.yml

```diff
@@ -57,9 +60,9 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           pip --version
-          pip install build
+          pip install hatch
```

Comment:
why do we need to install hatch explicitly? Doesnt build-backend in pyproject.toml handle this?

Reply:
i see no benefit of doing that. let's go with hatch